packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -1380,7 +1394,11 @@ function CheckoutLineItem( {
 			<span aria-labelledby={ itemSpanId } className="checkout-line-item__price">
 				{ shouldShowComparison ? (
 					<>
-						{ monthlyAmountDisplay } { translate( '/month' ) }
+						<LineItemPrice
+							actualAmount={ monthlyAmountDisplay }
+							crossedOutAmount={ hasIntroOffer ? originalMonthlyAmountDisplay : undefined }
```

Comment:
I wonder if this should really be limited to the `hasIntroOffer` case, or also shown for other kinds of discounts?

For example, I find this discrepancy a bit confusing:

![prorated-upgrade](https://github.com/user-attachments/assets/2615f983-7bb5-413d-b4fe-6d741e3cc678)

The above example is when the discount comes from upgrading to a higher plan, e.g. I started on a site that had Premium and then went to upgrade to Business.  (I'm not sure if that's a specific scenario that people can actually encounter during the initial experimental rollout or not, although in the long run it would be.)